core/api/src/main/java/org/trellisldp/api/MementoService.java

```diff
@@ -32,10 +32,13 @@
      * Create a new Memento for a resource, retrieved from a {@link ResourceService}.
      * @param resourceService the resource service.
      * @param identifier the identifier.
-     * @apiNote the default implementation of this method fetches a resource from a {@link ResourceService} that is
-     *          external to the Memento service. In the case that the two services are managed by the same persistence
-     *          layer, it may not be necessary to fetch a {@link Resource} from the persistence layer, in which case
-     *          this method can be overridden as a no-op method, e.g. {@code return completedFuture(null);}.
+     * @implSpec The default implementation of this method fetches a resource from a {@link ResourceService} that is
+     * external to the Memento service. 
+     * @implNotes In the case that the two services are managed by the same persistence layer, it may not be
+     * necessary to fetch a {@link Resource} from the persistence layer, in which case this method can be
+     * overridden as a no-op method, e.g. {@code return completedFuture(null);}.
+     * @apiSpec An implementation may choose to store a new Memento only when this method is called,
```

Comment:
There isn't a defined `@apiSpec` tag -- only `@apiNote`, `@implNote` and `@implSpec`.